charts/owncloud/values.yaml

```diff
@@ -17,29 +17,543 @@ nameOverride: ""
 fullnameOverride: ""
 
 owncloud:
-  domain: owncloud.chart.example
-  username: owncloud
-  password: owncloud
-  volume_apps: /mnt/data/apps
-  volume_config: /mnt/data/config
-  volume_files: /mnt/data/files
-  volume_root: /mnt/data
+  # -- Location of the access log.
+  # @default -- /dev/stdout
```

Comment:
The default is set automatically from the value. You just need to override it in some special cases with an `@default` line. Can be removed in most cases below as well.

Reply:
Fixed